src/CoreHealthMonitor/CoreHealthMonitor.cs

```diff
@@ -172,7 +176,7 @@ await using (FileStream stream = File.OpenRead(file))
 
     private bool IsIgnoredFile(string fileName)
     {
-        string[] patterns = _memoryDumpOptions.CurrentValue.IgnoreDumpPatterns;
+        string[] patterns = _memoryDumpOptions.Value.IgnoreDumpPatterns;
```

Comment:
Why the change from options monitor to options?

Reply:
We needed OptionMonitors when we were using secrets, because in case of a secret rotation, the new value would get picked up without a deployment. The container url is stable and never changes

Reply:
Until it does 😅
I agree that it might not but don't know why we would make it less flexible? Is there some cost to using `OptionsMonitor`?

Reply:
looks like we're not even using that part of the `OptionsMonitor` anywhere. We're basically using it like `IOptions`

Reply:
https://alirezafarokhi.medium.com/difference-between-ioptions-ioptionssnapshot-and-ioptionsmonitor-in-asp-netcore-587954bbcea